website/content/en/docs/scorecard-alpha/_index.md

```diff
@@ -0,0 +1,222 @@
+tests it will execute.  If a selector flag is not supplied, then all 
+the tests within the scorecard configuration file are executed.
+
+Tests are executed serially, one after the other, with test results 
+being aggregated by scorecard to present back to the end user.
+
+Here are some examples of supplying a selector flag to scorecard:
+```sh
+operator-sdk scorecard -o text --selector=test=basic-check-spec-test
+operator-sdk scorecard -o text --selector=suite=olm
+operator-sdk scorecard -o text --selector='test in (basic-check-spec-test,olm-bundle-validation-test)'
+```
+
+### Basic Operator
+
+| Test        | Description   | Test Name |
+| --------    | -------- | -------- |
+| Spec Block Exists | This test checks the Custom Resource(s) created in the cluster to make sure that all CRs have a spec block. | basic-check-spec-test |
+
+### OLM Integration
+
+| Test        | Description   | Short Name |
+| --------    | -------- | -------- |
+| OLM Bundle Validation | This test validates the OLM bundle manifests found in the bundle that is passed into scorecard.  If the bundle contents contain errors, then the test result output will include the validator log as well as error messages from the validation library.  See this [document][olm-bundle] for details on OLM bundles.| olm-bundle-validation-test |
+| Provided APIs have validation |This test verifies that the CRDs for the provided CRs contain a validation section and that there is validation for each spec and status field detected in the CR. | olm-crds-have-validation |
+| Owned CRDs Have Resources Listed | This test makes sure that the CRDs for each CR provided via the `cr-manifest` option have a `resources` subsection in the [`owned` CRDs section][owned-crds] of the CSV. If the test detects used resources that are not listed in the resources section, it will list them in the suggestions at the end of the test. | olm-crds-have-resources |
+| Spec Fields With Descriptors | This test verifies that every field in the Custom Resources' spec sections have a corresponding descriptor listed in the CSV.| olm-spec-descriptors-test |
+| Status Fields With Descriptors | This test verifies that every field in the Custom Resources' status sections have a corresponding descriptor listed in the CSV.| olm-status-descriptors-test |
+
+## Scorecard Output
+
+### JSON format
+
+Here is an example of the JSON format produced by a scorecard test:
+
+```json
+{
+  "metadata": {
+    "creationTimestamp": null
+  },
+  "log": "",
+  "results": [
+    {
+      "name": "basic-check-spec",
+      "description": "Custom Resource has a Spec Block",
+      "state": "pass"
+    }
+  ]
+}
+```
+
+### Text format
+
+Here is an example of the text format produced by a scorecard test:
+
+```
+	basic-check-spec                    : pass
+	CR: 
+	Labels: 
+```
+
+This output is structured according to the golang struct found
+[here][scorecard-struct].
+
+## Exit Status
+
```

Comment:
It may, something as: 

```
The scorecard result follows the Unix command standards and good practices. In this way, if all tests are executed successfully the command returns a 0, while if at least 1 test fails it will return the code value `1` that usually can be interpreted as an error code.
```